parsec/backend/postgresql/organization.py

```diff
@@ -276,7 +277,9 @@ async def create(
                     *_q_insert_organization(
                         organization_id=id.str,
                         bootstrap_token=bootstrap_token,
-                        active_users_limit=active_users_limit,
+                        active_users_limit=active_users_limit
+                        if active_users_limit is not ActiveUsersLimit.NO_LIMIT
+                        else None,
```

Comment:
Should not this case handled by `handle_interger` ?

Reply:
> We need to be careful when setting a custom type handler for integer to convert ActiveUsersLimit. I noticed that encoder (and decoder) are not supposed to return None (it may be handled differently by asyncpg) so we still need to check if the value is set to NO_LIMIT and set it to None by hand when performing an update or insert. I've added an assert for that.

We can't do that because `NO_LIMIT` is just `NULL` in the database. If our `encoder` returns `None` it will crash in `asyncpg` at some points.